torch/csrc/jit/passes/tensorexpr_fuser.h

```diff
@@ -17,7 +17,7 @@ TORCH_API void FuseTensorExprs(
     std::shared_ptr<Graph>& graph,
     size_t min_group_size = 2,
     bool add_composed_op = false,
-    bool fuse_to_dynamic_shapes = false);
+    bool fuse_to_dynamic_shapes = true);
```

Comment:
We shouldn't be changing the default as part of this PR i don't think

Reply:
Oh sorry it was a draft PR. Just updated it, and now it's ready for review! 

In this PR, aten::stack is not enabled for fusion yet. Will enable the fusion (w/wo dynamic shapes) in separate PRs.